docs/administration/support-package.md

```diff
@@ -0,0 +1,13 @@
+# Support Package
+
+Support packages are usefull for troubleshooting issue that occurs on OpenCTI platform.
+Administrators can request to create and download a support package that contains recent platform error logs and usage statistics.
+
+Support Package can be requested from "Settings > Support" menu.
+
+![Support package overview](./assets/support-package-overview.png)
+
+1. On a click on "Generate support package" button on the top right, the support package is requested.

```

Comment:
All the elements of the list have 1. as a number

Reply:
yes it's intended, it's a trick on markdown to not have to change number if you insert a new item in the middle